<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIIRS-tools Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="browserGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>File Browser</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="tsGroupBox">
           <property name="title">
            <string>Available timestamps:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QListView" name="tsListView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutMode">
               <enum>QListView::SinglePass</enum>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="viewMode">
               <enum>QListView::ListMode</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="fileGroupBox">
           <property name="title">
            <string>.tiff files:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListView" name="fileListView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="FigureGroupBox">
        <property name="title">
         <string>Figure</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="figureLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>109</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ActionsGroupBox">
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="CmGroupBox">
           <property name="title">
            <string>Cloud Mask</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="rCmBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Rapid CM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fireCmBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Fire CM</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="lstGroupBox">
           <property name="title">
            <string>LST</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QPushButton" name="monoLstBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Mono-window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="monoMaskedLstBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Mono-window (masked)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="nightGroupBox">
           <property name="title">
            <string>Night Mask</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QPushButton" name="naiveNmBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Naive</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="fireGroupBox">
           <property name="title">
            <string>Fire</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QPushButton" name="ActiveFireBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Active Fire</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="waterBodiesGroupBox">
           <property name="title">
            <string>Water Bodies</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QPushButton" name="wbBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Reflectance Test</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="statsGroupBox">
           <property name="title">
            <string>Statistics</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QPushButton" name="ndviBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>NDVI</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mlGroupBox">
           <property name="title">
            <string>Machine Learning</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QPushButton" name="cmMlBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Predict Cloud Mask</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveJpgBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save figure (jpg)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveTiffBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save composit (tiff)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
